code/modules/surgery/organs/filling_organ.dm

```diff
@@ -52,7 +52,7 @@
 	//updates size caps
 	if(!issimple(H) && H.mind && organ_sizeable)
 		var/athletics = H.mind?.get_skill_level(/datum/skill/misc/athletics)
-		var/captarget = storage_per_size + (athletics * 4) + (storage_per_size * organ_size) // Updates the max_reagents in case the organ size changes
+		var/captarget = max_reagents + (athletics * 4) + (5 + storage_per_size + (storage_per_size * organ_size)) // Updates the max_reagents in case the organ size changes
```

Comment:
this is wrong math i think, it doubles the reagent capacity due to it already having that max reagent calc on insert, regardless. I already did this my own way on my upcoming pr.